neuland/simulation/generator/R3BNeulandGenerators.h

```diff
@@ -9,32 +9,27 @@
 
 // NOTE: what's the purpose of this file? Just function definitions?
 
-inline auto create_muon_generator(TRandom3& random_gen)
+inline auto create_muon_generator(TRandom3& random_gen,
+                                  int num_dp,
+                                  double energy,
+                                  double x_pos,
+                                  double y_pos,
+                                  double z_pos)
```

Comment:
Better use `[ROOT::Math::XYZVector](https://root.cern.ch/doc/master/namespaceROOT_1_1Math.html#a91b80e54b44a65c90d60e5c8ff128746)` for x, y and z position.

The reason is they can be easily swapped and it's easy to make a mistake about x_pos, y_pos and z_pos.